imsim/meta_data.py

```diff
@@ -7,4 +7,4 @@
 if 'IMSIM_DATA_DIR' in os.environ: # pragma: no cover
     data_dir = os.environ['IMSIM_DATA_DIR']
 else:
-    data_dir = os.path.join(install_dir, 'imsim', 'data')
+    data_dir = os.path.join(imsim_dir, 'data')
```

Comment:
LGTM, Can you actually delete the line that sets install_dir since you aren't using it anymore here?

Or, is it used other places for the installation?

Reply:
OK.  Removed.